components/nifosg/nifloader.cpp

```diff
@@ -1030,6 +1030,78 @@ namespace NifOsg
             }
         }
 
+        class PreciseLeafDepthCullCallback : public osg::DrawableCullCallback{
```

Comment:
It's probably a good idea to declare this in a header as there are other places where things can become translucent. For example, when the chameleon effect is applied to an actor, they become translucent, so all of their drawables will need this callback adding.